Assets/Scripts/LightControllerScript.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LightControllerScript : MonoBehaviour
+{
+    public float actualSpeed = 10.0f;
+    public GameObject[] Waypoints;
+    public float distance; //on which distance you want to switch to the next waypoint
+
+    private int counter = 0;
+    private Vector2 velocity;
+    private Vector3 direction;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        Debug.Log("CURRENT POSITION: " + transform.position);
```

Comment:
Forgot about all of these Debug statements, they should be fine to leave for now